client/src/client-only-routes/show-certification.tsx

```diff
@@ -351,6 +357,7 @@ const ShowCertification = (props: ShowCertificationProps): JSX.Element => {
                     })
                   }}
                 </h4>
+                <h5 style={{ marginTop: '15px' }}>{{ completionTime }}</h5>
```

Comment:
This should be a `p` or `span` element, because headers are header for a section, and completionTime is part of the certification section, not the header of it.

Reply:
I was thinking more of adding a screen only text wrapped around a header, this is why I am leaning into asking. If we are adding `hN` for `freeCodeCamp` logo, are we going to add the same for Microsoft one?